src/core/Track.cpp

```diff
@@ -3051,3 +3054,25 @@ void TrackView::createTCOView( TrackContentObject * tco )
 	}
 	tco->selectViewOnCreate( false );
 }
+
+
+
+
+void TrackView::muteChanged()
+{
+	FadeButton * actind = getActivityIndicator();
+	if (actind)
+	{
+		if (m_track->m_mutedModel.value())
+		{
+			actind->setActiveColor(QApplication::palette().color(
+							QPalette::Active,
+							QPalette::Highlight));
```

Comment:
```suggestion
			actind->setActiveColor(
				QApplication::palette().color(QPalette::Active, QPalette::Highlight)
			);
```

Easier to read?

Reply:
Some other alternatives, any preference?

1. Same initial indentation level, put color() arguments on the same line

```suggestion
			actind->setActiveColor(QApplication::palette().color(
				QPalette::Active, QPalette::Highlight
			));
```

2. Start the function with an `if (!actind) { return; }` statement so the calls can be indented by one level less, put the entire argument to setActiveColor on it's own line

```suggestion
		actind->setActiveColor(
			QApplication::palette().color(QPalette::Active, QPalette::Highlight)
		);
```

Reply:
I don't feel like 1 is as readable. 2 is essentially the same as my suggestion. :)